_includes/header.html

```diff
@@ -13,7 +13,8 @@ <h1 class="hero-heading mb-5">
                     <span class="desc d-block"></span>
                 </h1>
                 <div class="text-center mb-5">
-                    <a href="#section-content" class="btn btn-primary btn-lg scrollto">Our Suggested Timeline</a>
+                    <a href="https://advanced-programming-sut-2023.github.io/" class="btn btn-primary btn-lg scrollto" style="margin-right:10px;">Spring 2023</a>
+                    <a href="https://advprog2020.github.io/" class="btn btn-primary btn-lg scrollto">Spring 2020</a>
```

Comment:
I think changing this section is not a good idea, and there are some problems with both buttons you added/modified:

```HTML
<a href="https://advanced-programming-sut-2023.github.io/" class="btn btn-primary btn-lg scrollto" style="margin-right:10px;">Spring 2023</a>
```
In this button, you may not use the `scrollto` class because it is meant to point to sections in the current page to be scrolled to with animation.

```HTML
<a href="https://advprog2020.github.io/" class="btn btn-primary btn-lg scrollto">Spring 2020</a>
```
Why do we need to change this button at all? It was supposed to scroll the user to the course's timeline and educational material. So, IMO, we don't need to change this button at all.

I think it's better to add an item to the navbar at `_includes/navbar.html` pointing to the new website rather than adding it here.